packages/react-native-gesture-handler/src/web/interfaces.ts

```diff
@@ -130,7 +130,9 @@ export interface ResultTouchEvent
 
 export interface PropsRef {
   onGestureHandlerEvent: () => void;
+  onGestureHandlerAnimatedEvent: () => void;
   onGestureHandlerStateChange: () => void;
+  onGestureHandlerTouchEvent: () => void;
```

Comment:
If we want to type it correctly, the new fields would have to be optional, so the old implementations are compatible (or we can pass no-ops there).

```suggestion
  onGestureHandlerEvent: () => void;
  onGestureHandlerAnimatedEvent?: () => void;
  onGestureHandlerStateChange: () => void;
  onGestureHandlerTouchEvent?: () => void;
```